content/install/kubernetes/microk8s.md

```diff
@@ -47,13 +37,19 @@ Wait for all the pods to start:
 microk8s.kubectl get pods -n space-cloud --watch
 ```
 
-The last step would be to port forward Space Cloud's ports.
+The last step would be to port forward Space Cloud's ports on localhost:
 
 ```bash
 microk8s.kubectl port-forward -n space-cloud gateway-0 4122:4122
 ```
 
-## Step 4: Open Mission Control
+Or if you are going to access microk8s remotely from another machine, use the following command instead (replacing YOUR_IP with its value):
```

Comment:
Can you do the same for `minikube` as well?

Reply:
We decided port 4122 access was redundant with port 30122 access.